price_feeds_poller.py

```diff
@@ -342,12 +397,36 @@ def main(args):
         w3.middleware_onion.add(middleware.simple_cache_middleware)
 
         network_gas_price = None
+        print("Gas price strategy: estimate_medium")
+      
+      elif network_gas_price is None:
+        w3.eth.set_gas_price_strategy(rpc_gas_price_strategy)
+        print("Gas price strategy: eth_gasPrice")
+      
       else:
         if network_gas_price == "estimate_medium" and w3.eth.chainId != 1:
```

Comment:
This is unreachable because above there is an `if network_gas_price == "estimate_medium":`, so this condition will never be true.

Reply:
True